# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/targets/angle_gpu_info_util/moz.build
# Commit: 754436d8b3a2
# Full Hash: 754436d8b3a2023b90eb247b3a70335e74f2fdf6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-09 09:14:23
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r f8fd6f0a5102 -r 754436d8b3a2 gfx/angle/targets/angle_gpu_info_util/moz.build
--- a/gfx/angle/targets/angle_gpu_info_util/moz.build	Fri May 08 18:39:28 2020 +0000
+++ b/gfx/angle/targets/angle_gpu_info_util/moz.build	Fri May 08 19:28:53 2020 +0000
@@ -6,18 +6,13 @@
 DEFINES['ANGLE_ENABLE_D3D9'] = True
 #DEFINES['ANGLE_IS_64_BIT_CPU'] = True
 #DEFINES['CERT_CHAIN_PARA_HAS_EXTRA_FIELDS'] = True
-#DEFINES['CHROMIUM_BUILD'] = True
-DEFINES['CR_CLANG_REVISION'] = '"357692-1"'
+DEFINES['CR_CLANG_REVISION'] = '"n332890-c2443155-1"'
 DEFINES['DYNAMIC_ANNOTATIONS_ENABLED'] = '0'
-#DEFINES['FULL_SAFE_BROWSING'] = True
 #DEFINES['NDEBUG'] = True
 DEFINES['NOMINMAX'] = True
-#DEFINES['NO_TCMALLOC'] = True
 DEFINES['NTDDI_VERSION'] = 'NTDDI_WIN10_RS2'
 #DEFINES['NVALGRIND'] = True
 #DEFINES['PSAPI_VERSION'] = '2'
-#DEFINES['SAFE_BROWSING_CSD'] = True
-#DEFINES['SAFE_BROWSING_DB_LOCAL'] = True
 DEFINES['UNICODE'] = True
 #DEFINES['USE_AURA'] = '1'
 #DEFINES['WIN32'] = True
@@ -53,7 +48,7 @@
 #    '-fcrash-diagnostics-dir=../tools/clang/crashreports',
 #    '-fdebug-compilation-dir',
 #    '-fmerge-all-constants',
-#    '-fmsc-version=1911',
+#    '-fmsc-version=1916',
 #    '-fno-standalone-debug',
 #    '-gcodeview-ghash',
 #    '-instcombine-lower-dbg-declare=0',
@@ -63,11 +58,17 @@
 #    '-Wextra-semi',
 #    '-Wheader-hygiene',
 #    '-Wimplicit-fallthrough',
+#    '-Wno-bitwise-conditional-parentheses',
+#    '-Wno-builtin-assume-aligned-alignment',
 #    '-Wno-builtin-macro-redefined',
 #    '-Wno-c++11-narrowing',
+#    '-Wno-deprecated-copy',
+#    '-Wno-final-dtor-non-final-class',
 #    '-Wno-ignored-pragma-optimize',
+#    '-Wno-implicit-int-float-conversion',
 #    '-Wno-missing-field-initializers',
 #    '-Wno-nonportable-include-path',
+#    '-Wno-sizeof-array-div',
 #    '-Wno-undefined-var-template',
 #    '-Wno-unneeded-internal-declaration',
 #    '-Wno-unused-parameter',
@@ -144,6 +145,7 @@
 #    '/Z7',
 #    '/Zc:inline',
 #    '/Zc:sizedDealloc-',
+#    '/Zc:twoPhase',
 #]
 
 SOURCES += [
@@ -173,13 +175,14 @@
 #    '/DEBUG:GHASH',
 #    '/FIXED:NO',
 #    '/INCREMENTAL:NO',
+#    '/lldignoreenv',
 #    '/OPT:ICF',
 #    '/OPT:NOLLDTAILMERGE',
 #    '/OPT:REF',
 #    '/pdbaltpath:%_PDB%',
 #    '/PDBSourcePath:C:/dev/angle/out',
 #    '/PROFILE',
-#    '/TIMESTAMP:1554613200',
+#    '/TIMESTAMP:1575176400',
 #    '/WX',
 #]
 